extensions/wikia/AssetsManager/config.php

```diff
@@ -13,6 +13,7 @@
 		'//resources/mediawiki/mediawiki.Uri.js',
 		'#group_banner_notifications_js',
 		'#group_ui_repo_api_js',
+		'//extensions/wikia/CookieSyncer/js/cookiesyncer.js',
```

Comment:
@mszabo-wikia is there any reason not to do that?

Reply:
I think it should be ok, maybe `oasis_noads_extensions_js` is a better place